detox/src/utils/easy-file-io.js

```diff
@@ -0,0 +1,13 @@
+const tempfile = require('tempfile');
+const fs = require('fs');
+
+function saveRawBase64Data(dataBase64, { filePath, fileSuffix }) {
+  const _filePath = filePath || tempfile(fileSuffix);
+  const data = Buffer.from(dataBase64, 'base64');
+  fs.writeFileSync(_filePath, data);
```

Comment:
How do you clean up temporary files after the test session ends?

Reply:
Any good sense to bother with Buffer conversions?
```js
fs.writeFileSync('hello.txt', 'SGVsbG8sIHdvcmxkIQ==', 'base64')
fs.readFileSync('hello.txt', 'utf8'); // 'Hello, world!'
```

Reply:
😻 😻 😻